### DISCUSSION TITLE
Custom File Size Validation Not Working

### DISCUSSION THREAD

**Discussion by elicordero111** [livewire/livewire#8783] [1. Help]
Greetings,

I'm experiencing issues with file validation. When I upload a file, it doesn't display the custom maximum size error message that I've set using `#[Validate('required|file|max:1024')]`. Instead, it shows the default error message indicating a limit of 12288 kilobytes.  For some reason, it's not validating against the custom value I've specified.

I would greatly appreciate any assistance you can provide.

**Comment by francoism90** [comment]
https://github.com/livewire/livewire/blob/597a2808d8d3001cc3ed5ce89a6ebab00f83b80f/config/livewire.php#L66

https://livewire.laravel.com/docs/uploads#file-validation

Are you uploading to S3?

**Reply by elicordero111** [reply]
No, I'm using local storage by default, this line of code from the FileUploadConfiguration file of the rule() function
`if (is_null($rules)) return ['required', 'file', 'max:12288'];`

may have something to do with it.
